metricbeat/module/elasticsearch/index/data_xpack.go

```diff
@@ -200,12 +194,6 @@ func addClusterStateFields(idx *Index, clusterState common.MapStr) error {
 		return errors.Wrap(err, "failed to get shards from routing table")
 	}
 
-	created, err := getIndexCreated(indexMetadata)
-	if err != nil {
-		return errors.Wrap(err, "failed to get index creation time")
-	}
-	idx.Created = created
```

Comment:
I think we can remove the `Created` field from the struct as well?

Reply:
Good point